src/scenic/simulators/carla/recording/__main__.py

```diff
@@ -0,0 +1,36 @@
+import os
+import subprocess
+import argparse
+import json
+
+parser = argparse.ArgumentParser(prog='scenic', add_help=False,
+                                 usage='scenic [-h | --help] [options] FILE [options]',
+                                 description='Sample from a Scenic scenario, optionally '
```

Comment:
This is misleading, since this entry point doesn't support most of Scenic's options. Please change `prog`, `usage`, and `description` to make clear this is a helper script for running multiple scenarios and collecting recordings.

(If you decide to keep this entry point. You could just get rid of it, or add the `--scenarios` functionality to the main Scenic entry point, which would allow you to drop the `subprocess` stuff.)